plugins/react-static-plugin-sharp/node.api.js

```diff
@@ -2,17 +2,19 @@ const path = require('path');
 const sharp = require('sharp');
 const fs = require('fs-extra');
 const klaw = require('klaw');
+const imagemin = require('imagemin');
+const imageminPngquant = require('imagemin-pngquant');
 
 const publicImagesDir = path.resolve(__dirname, '..', '..', 'public', 'images');
 const distImagesDir = path.resolve(__dirname, '..', '..', 'dist', 'images');
 
-const sizes = [{ height: 500, default: true }, { height: 700, name: 'lg' }, { height: 130, name: 'sm' }];
+const sizes = [{ width: 1200, default: true }, { width: 1600, name: 'lg' }, { width: 400, name: 'sm' }];
 
 export default pluginOptions => ({
   afterExport: async state => {
-    if (process.env.RELEASE_STAGE === 'production') {
-      await processImages();
-    }
+    // if (process.env.RELEASE_STAGE === 'production') {
+    //   await processImages();
```

Comment:
Need to keep this uncommented so the images are processed during the preview build

```suggestion
    await processImages();
```